### PR TITLE
Add support for Windows console

### LINKED ISSUE #5: Windows support
It seems that colors do not work in Windows.
In an ipython session:
image
In a python script:
image

Also, the  following example, that is stated in the readme, does not work, at least in my Windows box.
```python
msg = Printer()
with msg.loading("Loading..."):
    # Do something here that takes longer
    time.sleep(10)
msg.good("Successfully loaded something!")
```

It should be wrapped under the `__name__ == "__main__"` like this:

```python
if __name__ == "__main__":
    msg = Printer()
    with msg.loading("Loading..."):
        # Do something here that takes longer
        time.sleep(10)
    msg.good("Successfully loaded something!")
```
Otherwise you will get a `RuntimeError`:

```
An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
```

### PR DESCRIPTION
**Pull request by njsmith**
Modern Windows consoles have full ANSI escape support. Let's use it!

Fixes #5

### CODE REVIEW

**Comment by njsmith** [comment]
Unfortunately, I don't know a great way to test this in CI, because Windows consoles are weird built-into-the-OS-desktop things. But I tested it locally and it seems to work.

Other comments:
- I dropped Pocket PC support while I was in there. They haven't been sold since 2007 so it seems safe.
- I'm surprised that you don't use `isatty` to only enable color output on tty devices, like most programs that output color. Is there a particular reason? It'd be trivial to add that here too.

File: wasabi/util.py [hunk 1]
```diff
@@ -201,6 +201,53 @@ def can_render(string):
...
+        if flags & ENABLE_VIRTUAL_TERMINAL_PROCESSING:
+            return True
+        else:
+            return False
+    except OSError:
+        return False
+
 def supports_ansi():
     """Returns True if the running system's terminal supports ANSI escape
     sequences for color, formatting etc. and False otherwise. Inspired by
```
Comment (richardpaulhudson) [inline wasabi/util.py:253]: We should probably remove the comment about Django as it no longer applies.

**Comment by richardpaulhudson** [comment]
If there isn't already a Wasabi release planned in the near future, could we consider releasing a new minor version with this functionality and bumping the spaCy requirement?

File: wasabi/util.py (hunk 1 above)
Comment (njsmith) [reply wasabi/util.py]: Good point!

**Pull request merged**